probability/card_game.cpp

```diff
@@ -0,0 +1,185 @@
+/**
+ * @file
+ * @brief This is a simple card game
+ * @details
+ * This game is based off of https://www.codechef.com/problems/CARDS777 in which
+ * the player chooses a set number of red cards, blue cards and tokens. The user
+ * selects what color token they want and if that matches the color of the
+ * randomly generated card, the player recieves a point and if it does not
+ * match, the player gets no points. The games can happen until the player quits
+ * or until they have played all of their turns which is the number of red cards
+ * plus the number of blue cards. The token that is selected and the card that
+ * is generated is tossed.
+ * @author [Dhrubo Mondal](https://github.com/dhrubo28)
+ */
+#include <stdlib.h>
```

Comment:
This is a C header. Let's use `cstdlib` so we use a C++ header. :)
```suggestion
#include <cstdlib>
```